di_website/home/models.py

```diff
@@ -159,7 +169,73 @@ class Meta():
 
     parent_page_types = []  # prevent from being a child page
 
-    content_panels = Page.content_panels + [hero_panels()]
+    featured_publication = models.ForeignKey(
```

Comment:
Could you make this a foreign key to `Page` and in the `PageChooserPanel` point the target models to `['publications.PublicationPage','publications.ShortPublicationPage','publications.LegacyPublicationPage']` ? Right now users can only select one type of publication page.

Reply:
Done.